### PR TITLE
docs: Updated version history with 1.13.8 release notes.

### PR DESCRIPTION
**Pull request by cpakulski**
Updated version history with backports to 1.13.
Closing release 1.13 as it reaches EOL.

### CODE REVIEW

File: docs/root/version_history/v1.13.8.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+1.13.8 (January 15, 2021)
+=========================
+
+Changes
+-------
```
Comment (asraa) [inline docs/root/version_history/v1.13.8.rst:5]: should there be a space after Changes as in these release notes? 
https://github.com/envoyproxy/envoy/pull/14306/files

**Review by asraa — COMMENTED** [review]
Other than the formatting, LGTM!

File: docs/root/version_history/v1.13.8.rst (hunk 1 above)
Comment (cpakulski) [reply docs/root/version_history/v1.13.8.rst:5]: Thanks for catching it!

**Review by asraa — APPROVED** [review]
Thanks!

**Pull request merged**